packages/core/src/types/displayArea.ts

```diff
@@ -0,0 +1,25 @@
+type InitialDisplayArea = {
```

Comment:
These should all be two element arrays for consistency.

Reply:
this change is now made